package.json

```diff
@@ -35,10 +35,6 @@
         {
           "command": "assay.openInDiffTool",
           "when": "false"
-        },
-        {
-          "command": "assay.welcome",
-          "when": "false"
```

Comment:
this was a deliberate delete?

Reply:
Yes, this allows the documentation to be accessible via the command palette